ibis/common/pretty.py

```diff
@@ -49,6 +49,24 @@ def show_sql(
       t0.a * 2 AS c
     FROM t AS t0
     """
+    print(sql(expr, dialect=dialect), file=file)
+
+
+def sql(expr: ir.Expr, dialect: str | None = None) -> str:
```

Comment:
I'm a bit worried about this being conflated with `Backend.sql` - could we call it `to_sql` instead?

Reply:
Sure.